product_docs/docs/epas/11/installing/index.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: "Installing EDB Postgres Advanced Server"
+navTitle: "Installing"
+redirects:
+ - /epas/11/epas_inst_linux/
+ - /epas/11/epas_inst_linux/installing_epas_using_edb_repository/
+navigation:
+- linux_x86_64
+- linux_ppc64le
+- windows
+- linux_install_details
+- windows_install_details
+---
+## [Linux x86-64 (amd64)](linux_x86_64)
+### Red Hat Enterprise Linux (RHEL) and derivatives
+- [RHEL 8](linux_x86_64/epas_rhel_8), [RHEL 7](linux_x86_64/epas_rhel_7)
+- [Oracle Linux (OL) 8](linux_x86_64/epas_rhel_8), [Oracle Linux (OL) 7](linux_x86_64/epas_rhel_7)
+- [Rocky Linux 8](linux_x86_64/epas_other_linux_8)
+- [AlmaLinux 8](linux_x86_64/epas_other_linux_8)
+- [CentOS 7](linux_x86_64/epas_centos_7)
+### SUSE Linux Enterprise (SLES) 
+- [SLES 15](linux_x86_64/epas_sles_15), [SLES 12](linux_x86_64/epas_sles_12)
+### Debian  and derivatives
+- [Ubuntu 20.04](linux_x86_64/epas_ubuntu_20), [Ubuntu 18.04](linux_x86_64/epas_ubuntu_18)
+- [Debian 11](linux_x86_64/epas_debian_11), [Debian 10](linux_x86_64/epas_debian_10)
+## [Linux on IBM Power (ppc64le)](linux_ppc64le)
+### Red Hat Enterprise Linux (RHEL) 
+- [RHEL 8](linux_ppc64le/epas_rhel_8), [RHEL 7](linux_ppc64le/epas_rhel_7)
+### SUSE Linux Enterprise (SLES) 
+- [SLES 15](linux_ppc64le/epas_sles_15), [SLES 12](linux_ppc64le/epas_sles_12)
+## [Windows](windows)
```

Comment:
I think we can have Just Windows as title and underneath a title with link to the install topic

Reply:
I agree we need to do something slightly differently here. We also got his comment from Matt on the EPRS PR where this new approach was implemented: 
> I think having the section header as the hyperlink to the Windows installer is a bit weird and confusing. Although it may be somewhat redundant, I think there should be at least a simple sentence following the header with the hyperlink.

@dwicinas, let's use this approach for the Windows links:
```
## Windows
- [Windows Server 2019](windows)
```

It matches the platform compatibility page and doesn't look so weird. I think MTK is the only other product where we need to make this change.

Reply:
I made this change and will do the same for the other EPAS versions